department-of-veterans-affairs/va.gov-cms
# VACMS-1089 Get medialibrary widget working properly.

## Description

See #1089 

THis PR essentially reverts the views to what media_library defines, then adds our customizations back to it.

## Testing done


## Screenshots


## QA steps

As user _uid_ with _user_role_
1. As a content admin got to /node/add/documentation_page
1. Then scroll down to "Main Content" and click the "Add Content block" button.
1. Select the "EMBEDDED IMAGE" block type
1. Click "Add media" button.
  - [x] Validate that the media widget appears and only contains media of type images (no movies, no documents)
![image](https://user-images.githubusercontent.com/5752113/77697530-655abb80-6f85-11ea-812a-9e5ea5eca925.png)

  - [x] Validate that the filters and disply toggle match 
![image](https://user-images.githubusercontent.com/5752113/77697507-5ecc4400-6f85-11ea-9ae3-fc3b508b640f.png)

5. Add another content block, by clicking "Add content block" 
6. Scroll down and select "LINK TO FILE OR VIDEO"
7.  Click "Add media" button.
   - [x] Validate that when the "Document" tab is selected, only documents show.
   - [x] Validate that when the "Image" tab is selected, only images show.
   - [x] Validate that when the "Video" tab is selected, only videos show show.
   - [x] Validate that name serach filter works by typing in "map" and then clicking "Apply filters"  You should only see maps.
   - [x] Validate that owner works by selecting "VA Butler health care while on the image tab and only a few results should be returned.
  - [x] Validate that selecting media and clicking "Insert selected" actually selects the media and inserts it into the node.
